### DISCUSSION TITLE
Qt 5.15.14 and OpenSSL 3.x

### DISCUSSION THREAD

**Discussion by LumitoLuma** [sandboxie-plus/Sandboxie#3943] [General]
Thanks to @offhub, I have ready an updated Qt build script for 5.15.14. It now uses OpenSSL 3.x, which is [supported from Qt 5.15.13](https://bugreports.qt.io/browse/QTBUG-107729?focusedId=716815&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-716815).

@offhub has also created a repo with a build scripts for these new OpenSSL builds: https://github.com/offhub/openssl-builds.

To update to this build, modification of the get_openssl.cmd script is required.

**Comment by offhub** [comment]
#3994
